src/EnergyPlus/PluginManager.cc

```diff
@@ -55,41 +61,355 @@
 #include <EnergyPlus/PluginManager.hh>
 #include <EnergyPlus/UtilityRoutines.hh>
 
-#if LINK_WITH_PYTHON == 1
-#ifdef _DEBUG
-// We don't want to try to import a debug build of Python here
-// so if we are building a Debug build of the C++ code, we need
-// to undefine _DEBUG during the #include command for Python.h.
-// Otherwise it will fail
-#undef _DEBUG
-#include <Python.h>
-#define _DEBUG
-#else
-#include <Python.h>
-#endif
-#endif
```

Comment:
We are in PluginManager, which is part of the EnergyPlus shared library.  The primary design change in this PR results in an EnergyPlus shared library that does not have a build time dependency on the Python DLL.  So we no longer import the Python header or hard link to the Python shared library functions.